libraries/common/stream.effekt

```diff
@@ -102,9 +102,20 @@ def exhaustively[A] { program: () => A / stop } { action: A => Unit }: Unit =
     }
     go()
   } with stop {
-    def stop() = ()
+    ()
   }
```

Comment:
This confused me like three different times while reading the code -- I don't think we should necessarily rely on the fact that this more explicit `def stop() = ...` syntax is allowed when using a singleton `effect`...

Reply:
This is a left-over from an earlier version, sorry.